### ISSUE TITLE
Just a little bug

### ISSUE THREAD

**Issue by Rasmur003** [ZuzAser80/EldritchArcaneWorkings#4]
Its that when your mod is installed, that items dont automatically stack together. So they just go into a new spot in the inventory.
Its nothing really troublesome i just wanted to let you now.

**Comment by ZuzAser80** [comment]
Uh, thanks, but witch items exactly? Vanilla ones or the ones added by my mod? Thanks for letting me know though.

**Comment by Rasmur003** [comment]
i havent tested it out on many items, but it seems to be any item vanilla or by your mod

**Comment by ZuzAser80** [comment]
Oh. That's weird, thanks, ill test more.

**Comment by TalEdd** [comment]
I just realized this was happening to my inventory also. Was just going to report it. Also can  the flower names be fixed, If its not too much work?
![image](https://user-images.githubusercontent.com/63492921/193410048-0167ec75-5303-452c-a40a-aecfb08abd62.png)

**Comment by ZuzAser80** [comment]
Oh. It looks like i skipped another entry in lang file. My bad, i will fix it in the next update.

**Comment by ZuzAser80** [comment]
I'm uploading the debugged v1.0.2.4, it should be fixed. If not, let me know :) Thanks

**Comment by Rasmur003** [comment]
The items still are not stacking

**Comment by ZuzAser80** [comment]
Okay. I will find the issue and fix it soon. Thank you for feedback.

**Comment by ZuzAser80** [comment]
I think i have fixed it in the newest version, v1.0.3, though i'm not entirely sure.

**Comment by Kuggle01** [comment]
I've encountered this bug too, it seem every item will not stack when first entering the inventory

**Comment by ZuzAser80** [comment]
Kuggle01, do you have v1.0.3 or do you  have lesser version?

**Comment by Rasmur003** [comment]
i dont think you uploaded the file. In the mod page its says upadated 5 days ago

**Comment by Daihok41** [comment]
This is still happening with the v1.0.4 for 1.19.2
